docs/admin/hardware.md

```diff
@@ -1,113 +1,136 @@
-this approach usually requires high-density storage servers.
+As illustrated in the figure below, a DAOS storage system can be
+deployed in two different ways:
 
+-   **Pooled Storage Model**: The DAOS servers can run on dedicated
+    storage nodes in separate racks. This is a traditional pool model
+    where storage is uniformly accessed by all compute nodes. In order
+    to minimize the number of I/O racks and to optimize floor space,
+    this approach usually requires high-density storage servers.
 
-While DAOS is currently deployed following the pooled model, active research is
-conducted to efficiently support a hyper-converged model as well.
+-   **Hyper-converged Storage Model**: In this model, the storage nodes
+    are integrated into compute racks and can be either dedicated or
+    shared nodes. The DAOS servers are thus massively distributed, and
+    storage access is non-uniform and must take locality into account.
+    This model is common in hyper-converged infrastructure.
 
-![](./media/Fig_073.png)
+While DAOS is mostly deployed following the pooled model, active
+research is conducted to efficiently support the hyper-converged model
+as well.
 
-## Processor Requirements
+Processor Requirements
+----------------------
 
 DAOS requires a 64-bit processor architecture and is primarily developed
-on Intel x86\_64 architecture. The DAOS software and the libraries it depends
-on (e.g., ISA-L, SPDK, PMDK, and DPDK) can take advantage of Intel SSE
-and AVX extensions.
+on Intel x86\_64 architecture. The DAOS software and the libraries it
+depends on (e.g., ISA-L[^1], SPDK[^2], PMDK[^3], and DPDK[^4]) can take
+advantage of Intel SSE[^5] and AVX[^6] extensions.
 
-Some success was also reported by the community on running the DAOS client
-on 64-bit ARM processors configured in Little Endian mode. That being said,
-ARM testing is not part of the current DAOS CI pipeline and is thus not
-validated on a regular basis.
+The DAOS client is regularly tested on 64-bit ARM processors configured
+in Little Endian mode. The same build instructions that are used for
+x86\_64 are applicable for ARM builds as well. DAOS and its dependencies
+will make the necessary adjustments automatically in their respective
+build systems for ARM platforms.
 
-## Network Requirements
+Network Requirements
+--------------------
 
-The DAOS network layer relies on libfabrics and supports OFI providers
-for Ethernet/sockets, InfiniBand/verbs and RoCE.
-An RDMA-capable fabric is preferred for better performance. DAOS can support
-multiple network interfaces by binding different engines and different client
-processes to individual network cards.
+The DAOS data plane relies on OFI libfabrics[^7] and supports OFI
+providers for Ethernet/sockets and InfiniBand/verbs. An RDMA-capable
+fabric is preferred for better performance. DAOS can support multiple
+rails by binding different instances of the DAOS server to individual
+network cards.
```

Comment:
please keep the original text - "rails" have a special meaning that usually does not apply here, and it *is* the engine and not the server that is meant here.

Reply:
still referring to "rails".... which isn't how most DAOS systems are deployed: most DAOS systems have their multiple network interfaces connected to the same "rail", not to multiple "rails". so the original text that simply says "multiple network interfaces" is the more general and more appropriate description.